src/app/shell/workshop-details/side-menu/side-menu.component.html

```diff
@@ -1,7 +1,13 @@
 <div class="container" fxLayout="column"
  [fxLayout.lt-lg]="(isMobileScreen$ | async) ? 'column' : 'row wrap'"
  [fxLayoutAlign]="(isMobileScreen$ | async) ? 'none' : 'center start'">
-  <app-actions *ngIf="role !== Role.provider" [role]="role" [workshop]="workshop"></app-actions>
+
```

Comment:
try to follow general styleguides of the application and fix indentations

Reply:
done